Header header
float64 steering_torque
float64 brake_pressure
float64 throttle_fraction
uint8 gear
int16 steering_int
uint16 brake_int
uint16 throttle_int
uint64 bytes_written
float64 timestamp


